app/main.py

```diff
@@ -1623,21 +1623,18 @@ def add_or_update_emailoctopus_contact(list_id, email, firstname, lastname, tags
 @app.route("/hubspot_webhook", methods=["POST"])
 def hubspot_webhook():
     body = request.get_json()
```

Comment:
Why have this outside the try/except block and then inside it?

Reply:
oversight, i forgot to remove it